cirq-core/cirq/contrib/quimb/Contract-a-Grid-Circuit.ipynb

```diff
@@ -402,7 +404,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "ccq.tensor_expectation_value(circuit, ZZ)"
+    "# ccq.tensor_expectation_value(circuit, ZZ)"
```

Comment:
Is this intentional? Why not remove?

Reply:
This was a temporary change to check if that call causes CI failure in https://github.com/quantumlib/Cirq/actions/runs/7894234524/job/21545039838.
It indeed did, because the CI passed for 125178197b15b5c12594e190a2caf7e4e3be13ed.
I have also run the notebook in a stock colab where the last call crashed with out-of-memory.
Testing with a pre-release cirq works, because it has a pinned quimb version.

I am reverting the notebook to its mainline version and excluding it from tests with stable cirq.